src/crud.js

```diff
@@ -0,0 +1,24 @@
+export const list = document.querySelector('.score-list');
+export const scores = JSON.parse(localStorage.getItem('scoreList')) || [];
+
+export function createScore(name, score) {
```

Comment:
* Kindly use `arrow functions` instead of the `function` keyword as part of the learning objective of this project is to learn how to use proper ES6 syntax which includes the use of ES6 `arrow function` `import` `export` `const` `let` etc.[Refer to project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md#:~:text=Learn%20how%20to%20use%20proper%20ES6%20syntax.)